docs: enabled services and machine readable status updates (#2635)
canonical/ubuntu-pro-client: docs/explanations/status_columns.md

## Machine-readable output

### Status API functions

Some status information can be obtained from API functions. These include:

- Is the machine currently attached to an Ubuntu Pro subscription? See [u.pro.status.is_attached.v1](references/api:u.pro.status.is_attached.v1)
- Which Ubuntu Pro services are currently enabled on the machine? See [u.pro.status.enabled_services.v1](references/api:u.pro.status.enabled_services.v1)

> If you need status related information for which there is not an API function, then `pro status --format=json` (described below) is the recommended machine-readable interface.

Reviewer: ```suggestion
If you need status-related information for which there is no API function, then `pro status --format=json` (described below) is the recommended machine-readable interface.
```

```

Some particularly important attributes in the output include:

> * `attached`: This boolean value indicates whether this machine is attached to an Ubuntu Pro account (Starting with version 28.1, we recommend using the [u.pro.status.is_attached.v1](references/api:u.pro.status.is_attached.v1) API instead). This does not tell you if any particular service (e.g. `esm-infra`) is enabled. You must check the individual service item in the `services` list for that status (described below).

Reviewer: Since it's not really an aside, I think this is better off as a new sentence rather than being in brackets (same with the one just below at line 208).

Reviewer 2: Addressed
